docs/pressable.md

```diff
@@ -161,6 +161,22 @@ Sets additional distance outside of element in which a press can be detected.
 | ---------------------- |
 | [Rect](rect) or number |
 
+### `onHoverIn`
+
+Called when the hover is activated to provide visual feedback.
+
+| Type                                                |
+| --------------------------------------------------- |
+| ({ nativeEvent: MouseEvent }) => void |
```

Comment:
```suggestion
| ({ nativeEvent: [MouseEvent](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent) }) => void |
```

Let's add the link for returned type for the clarity that it mimics the native web event:
* https://github.com/facebook/react-native/blob/main/Libraries/Types/CoreEventTypes.d.ts#L111